### ISSUE TITLE
Update jquery

### ISSUE THREAD

**Issue by maximest-pierre** [lbryio/lbry.io#382]
The version of jquery that is bundle is really outdated. The version bundle is 2.1.3 and the current one is 3.3.1.

An alternative to using a bundled library is to use a CDN to deliver the appropriate dependency.

**Comment by lyoshenka** [comment]
What is the advantage of the newer jquery?

On Sat, Feb 17, 2018 at 10:04 PM, Maxime St-Pierre <notifications@github.com

**Comment by maximest-pierre** [comment]
There seems to be some speed improvement with some function and also better support for form and ajax. There is also some security patch. Before doing any changes to JQuery, we also must check that we don't break any functionality.

**Comment by ykris45** [comment]
#400
